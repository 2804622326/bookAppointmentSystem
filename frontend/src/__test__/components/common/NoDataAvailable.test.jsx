import React from 'react';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import NoDataAvailable from '../../../components/common/NoDataAvailable.jsx';

test('displays error message when provided', () => {
  const { asFragment } = render(
    <NoDataAvailable dataType="items" errorMessage="oops" />
  );
  expect(screen.getByText(/oops/i)).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});
