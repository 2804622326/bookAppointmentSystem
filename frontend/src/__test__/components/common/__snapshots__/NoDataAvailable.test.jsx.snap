// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays error message when provided 1`] = `
<DocumentFragment>
  <div
    class="text-center mt-5"
  >
    <h5>
      No items available at the moment
    </h5>
    <p
      class="text-danger"
    >
      oops
    </p>
  </div>
</DocumentFragment>
`;
